pycti/connector/opencti_connector_helper.py

```diff
@@ -1556,6 +1556,8 @@ def send_stix2_bundle(self, bundle: str, **kwargs) -> list:
         :type entities_types: list, optional
         :param update: whether to updated data in the database, defaults to False
         :type update: bool, optional
+        :param bypass_split: @deprecated [>=6.3 & <6.6] use to prevent splitting of the bundle. Not used anymore
```

Comment:
```suggestion
        :param bypass_split: use to prevent splitting of the bundle. This option has been removed since 6.3.2 and is no longer used.
```